java/runtime/src/main/java/org/ray/runtime/objectstore/ObjectStoreProxy.java

```diff
@@ -36,29 +30,26 @@
   private static final byte[] UNRECONSTRUCTABLE_EXCEPTION_META = String
       .valueOf(ErrorType.OBJECT_UNRECONSTRUCTABLE.getNumber()).getBytes();
 
+  private static final byte[] TASK_EXCEPTION_META = String
+      .valueOf(ErrorType.TASK_EXCEPTION.getNumber()).getBytes();
+
   private static final byte[] RAW_TYPE_META = "RAW".getBytes();
 
-  private final AbstractRayRuntime runtime;
+  private final WorkerContext workerContext;
 
-  private static ThreadLocal<ObjectStoreLink> objectStore;
+  private final ObjectInterfaceProtocol objectInterface;
 
-  public ObjectStoreProxy(AbstractRayRuntime runtime, String storeSocketName) {
-    this.runtime = runtime;
-    objectStore = ThreadLocal.withInitial(() -> {
-      if (runtime.getRayConfig().runMode == RunMode.CLUSTER) {
-        return new PlasmaClient(storeSocketName, "", 0);
-      } else {
-        return ((RayDevRuntime) runtime).getObjectStore();
-      }
-    });
+  public ObjectStoreProxy(WorkerContext workerContext, ObjectInterfaceProtocol objectInterface) {
+    this.workerContext = workerContext;
+    this.objectInterface = objectInterface;
   }
 
   /**
    * Get an object from the object store.
    *
-   * @param id Id of the object.
+   * @param id        Id of the object.
```

Comment:
nit: ident?  similar below.

Reply:
@kfstorm you can configure Intellij IDEA to use google style.